src/Servers/Kestrel/Transport.Sockets/src/SocketConnectionListener.cs

```diff
@@ -117,28 +119,33 @@ internal void Bind()
                 {
                     Debug.Assert(_listenSocket != null, "Bind must be called first.");
 
+
                     var acceptSocket = await _listenSocket.AcceptAsync(cancellationToken);
 
                     // Only apply no delay to Tcp based endpoints
+                    var noDelay = false;
+// REVIEW: should this move to inside of _contextFactory.Create??
```

Comment:
I think since SocketConnectionContextFactory.Create() takes a Socket, there's no reason to. They could set Socket.NoDelay themselves before calling Create().